regression/cbmc-java/exceptions19/exception_handler.j

```diff
@@ -0,0 +1,25 @@
+.class exception_handler
+.super java/lang/Object
+
+.method public <init>()V
+        aload_0
+        invokevirtual java/lang/Object/<init>()V
+        return
+.end method
+
+.method public f()V
+        .limit stack 4
+        .limit locals 4
+
+      begin:
+        new java/lang/Exception
+        dup
+        invokespecial java/lang/Exception.<init>()V
+        athrow
+      start:
+        nop
```

Comment:
@pkesseli is probably right, having someting like
```
start:
 iconst_0
 astore_1
end:
 astore1
...
```
is better, then `nop` integration could become a PR to master